[2020-12-24 23:10:09,612] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: data-cleaning-and-integration.store_data 2020-12-24T21:05:25.907747+00:00 [queued]>
[2020-12-24 23:10:09,622] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: data-cleaning-and-integration.store_data 2020-12-24T21:05:25.907747+00:00 [queued]>
[2020-12-24 23:10:09,622] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-12-24 23:10:09,622] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-12-24 23:10:09,622] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-12-24 23:10:09,628] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): store_data> on 2020-12-24T21:05:25.907747+00:00
[2020-12-24 23:10:09,631] {standard_task_runner.py:53} INFO - Started process 7431 to run task
[2020-12-24 23:10:09,676] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: data-cleaning-and-integration.store_data 2020-12-24T21:05:25.907747+00:00 [running]> 1.0.0.127.in-addr.arpa
[2020-12-24 23:10:09,704] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-12-24 23:10:09,707] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=data-cleaning-and-integration, task_id=store_data, execution_date=20201224T210525, start_date=20201224T211009, end_date=20201224T211009
[2020-12-24 23:10:19,609] {logging_mixin.py:112} INFO - [2020-12-24 23:10:19,608] {local_task_job.py:103} INFO - Task exited with return code 0
